sdk/src/main/java/com/ifountain/opsgenie/client/model/beans/BaseDataObj.java

```diff
@@ -0,0 +1,24 @@
+package com.ifountain.opsgenie.client.model.beans;
+
+public class BaseDataObj extends Bean{
```

Comment:
What's the utility of this object. Given its seems to encapsulate `Id` only. That'd allow us to not change `BaseResponseWithId` no?